Bottom: 792742960272f8ffa4fe3aef105f7dd780be12e8
Top:    36f20b3ce8b9322b02f5b1dcf1983d5cfcc397ff
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 18:03:31 +0800

Refresh of autofs-5.0.7-try-and-cleanup-after-dumpmaps.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 1d0b64e..44be46b 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2152,6 +2152,7 @@ int main(int argc, char *argv[])
 
 		lookup_nss_read_master(master_list, 0);
 		master_show_mounts(master_list);
+		master_kill(master_list);
 		exit(0);
 	}
